.rubocop.yml

```diff
@@ -71,3 +71,6 @@ RSpec/InstanceVariable:
 RSpec/SpecFilePathFormat:
   Exclude:
     - 'spec/routes/**/*.rb'
+
+Sequel/ColumnDefault:
+  Enabled: false
```

Comment:
Why do we need this? Removing this and running rubocop on my local didn't generate any errors.

Reply:
Huh, it definitely breaks for me with the migration I added.

Reply:
.....I wonder if adding `null: false` "fixed" it